### PR TITLE
syft/0.100.0-r1: cve remediation

### PR DESCRIPTION
**Pull request by chainguardian**
syft/0.100.0-r1: fix GHSA-9763-4f94-gfch

### CODE REVIEW

**Review by hectorj2f — DISMISSED** [review]

**Comment by hectorj2f** [comment]
I had to rebase the package to get the latest changes from main.

**Comment by hectorj2f** [comment]
To fix the update ci job, I updated Syft to latest version.

File: syft.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package:
   name: syft
-  version: 0.100.0
-  epoch: 1
+  version: 0.101.0
+  epoch: 2
```
Comment (luhring) [inline syft.yaml:4]:
```suggestion
  epoch: 0
```

File: syft.yaml [hunk 2]
```diff
@@ -13,9 +13,13 @@ environment:
...
   - uses: git-checkout
     with:
+      expected-commit: 297ece69045d6d73a184ed955a99d90769b1cb45
       repository: https://github.com/anchore/syft
       tag: v${{package.version}}
-      expected-commit: a16a4ad6c993aaaa1ec502667c215b0e8c8e51ff
+
+  - uses: go/bump
+    with:
+      deps: github.com/cloudflare/circl@v1.3.7
```
Comment (luhring) [inline syft.yaml:22]: I think you can drop this now

**Review by luhring — CHANGES_REQUESTED** [review]

**Comment by ajayk** [comment]
Lets Drop the whole PR the bot should get the update

**Pull request closed without merging**